From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Wed, 26 Oct 2022 13:48:04 -0400
Subject: [PATCH] pulsecore: static_assert -> _Static_assert for glibc 2.12
 compatibility

---
 src/pulsecore/shm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pulsecore/shm.c b/src/pulsecore/shm.c
index e464f6bce..5220cc90d 100644
--- a/src/pulsecore/shm.c
+++ b/src/pulsecore/shm.c
@@ -94,7 +94,7 @@ struct shm_marker {
 };
 
 // Ensure struct is appropriately packed
-static_assert(sizeof(struct shm_marker) == 8 * 5, "`struct shm_marker` is not tightly packed");
+_Static_assert(sizeof(struct shm_marker) == 8 * 5, "`struct shm_marker` is not tightly packed");
 
 static inline size_t shm_marker_size(pa_mem_type_t type) {
     if (type == PA_MEM_TYPE_SHARED_POSIX)
-- 
2.37.3

